src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String WORD_SEPARATOR = " ";
+    private static final String BY_WORD_BOUNDARIES = "\\W+";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, getResultString(fromFileName));
```

Comment:
```suggestion
       String fileContent = getResultString(fromFileName);
       String report = createReport(fileContent);
        writeToFile(toFileName, report );
```

Reply:
Please refactor your code to follow this structure, this approach will add flexibility to the code. For example, you want to reuse this class methods to just read files, or just generate reports